code-challenges.test.js

```diff
@@ -21,11 +33,31 @@ const fibLength2 = 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
 // b) Create the function that makes the test pass.
+// Pseudo: create a function called fibSequence that takes in a number greater than 2 and returns the an array of the same length containing the numbers following the rules of the Fibonacci sequence.
+// Within the function create an array called fibArray that contains the values [1,1]. Then create a for loop that iterates through fibArray while adding the current number and the previous number together but only when the index is at 2. The results will then be pushed into fibArray. The function will then return the desired Fibonacci sequence in fibArray.
+// const fibSequence = (number) => {
```

Comment:
AWESOME LOGIC! This is GREAT code though, there is nothing wrong with this but when you say you are looking for what makes it competent or things to look out for this is what I am thinking. 
Thinking competent: Refactor to a HOF and minimizing it to a one liner (Per feedback)